lib/robots/dor_repo/ocr/fetch_files.rb

```diff
@@ -26,19 +26,15 @@ def copy_files
         def abbyy_path(filename)
           # NOTE: if files of type "file" were allowed here we would have to
           # deal with file hierarchy (subdirectories)
-          Pathname.new(File.join(abbyy_input_path, filename))
-        end
-
-        def abbyy_input_path
-          @abbyy_input_path ||= Dor::TextExtraction::Ocr.new(cocina_object:, workflow_context: workflow.context).abbyy_input_path
+          Pathname.new(File.join(ocr.abbyy_input_path, filename))
```

Comment:
refactor: we do not need to initialize the `Ocr` class again, we already have it in a helper method below... which makes it not necessary to have an entire method to get the path